tests/unit/test_extractor.py

```diff
@@ -415,6 +405,33 @@ def test_upload_files_from_abfss_cognite_error(
     test_extractor.logger.error.assert_called_once()
 
 
+def test_retrieve_external_ids_from_lakehouse(test_extractor, mocker):
+    # Mock DeltaTable to return a pyarrow table with the externalId column
+    df = pd.DataFrame({"externalId": ["id1", "id2", "id1", "id1"]})
```

Comment:
is the repetition of "id1" intentional? It doesn't seem to be relevant to the test execution.

Reply:
The goal of this test is to ensure that the method successfully returns unique values.  The 3rd "id1" might not be necessary, but I want to ensure duplicates are removed.

Reply:
it might be more clear if the 3rd value different from the first value "id1" to make abundantly clear that the 1st  item in the array was returned, not the 3rd..